dist/freeciv-qt.appdata.xml.in

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
-    <id type="desktop">net.longturn.freeciv_qt</id>
+    <id type="desktop">net.longturn.freeciv21.qt</id>
```

Comment:
freeciv21_qt

Reply:
If going with the hierarchy model there, I would suggest using `.client_qt` at the end to follow along with `.modpack_qt`

Reply:
I think I will rename the client to `freeciv21_client`. Everything is Qt, no need to specify.